src/javascript/string-locator-instawp.js

```diff
@@ -0,0 +1,115 @@
+( function ( wp, $ ) {
```

Comment:
The JavaScript code here adds a reliance on jQuery, but the other pieces of JavaScript uses vanilla JS to reduce dependencies.